poetry.lock

```diff
@@ -253,6 +253,46 @@ packaging = "*"
 six = ">=1.9.0"
 webencodings = "*"
 
```

Comment:
ty for checking in poetry.lock lol (diego told me about this)

Reply:
yeah, there was a weird bug with other packages. I guess we just need to call `poetry add <package>` instead of doing it manually in pyproject.toml